Fix inclusion of header files to avoid compilation problems
with recent versions of gcc.
--- a/vbrfixc/vbrfix.cpp
+++ b/vbrfixc/vbrfix.cpp
@@ -16,6 +16,7 @@
  ***************************************************************************/
 #include "vbrfix.h"
 #include "arpa/inet.h"
+#include <cstring>
 //#include "stdafx.h"//include in mfc version
 int getXHpos(int ver,int chan);
 VbrFix::VbrFix(){
